app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -156,8 +157,9 @@ class ProfileChooserFragmentTest {
   fun testProfileChooserFragment_afterVisitingHomeActivity_showsJustNowText() {
     val data = profileTestHelper.initializeProfiles()
     launch<ProfileChooserActivity>(createProfileChooserActivityIntent()).use {
+      // Waiting for the simulation that HomeActivity is visited once.
```

Comment:
```suggestion
      // Note that this wait is needed to simulate HomeActivity being visited by ensuring a profile was previously logged in.
```

(may require line wrapping). Please copy elsewhere (as needed).

Reply:
done